containers/roles/foreman/files/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+
+/usr/bin/wait_on_postgres.py
+
+foreman-rake db:migrate
+echo $SEED_ADMIN_PASSWORD
+SEED_ADMIN_PASSWORD=$SEED_ADMIN_PASSWORD foreman-rake db:seed
+foreman-rake apipie:cache:index
```

Comment:
It is, there is a strategy for doing this in the build phase via a sqlite database. Would be awesome if we could eliminate the need for a DB.